persistence/src/test/java/cz/muni/fi/pa165/ManufacturerDAOTest.java

```diff
@@ -119,4 +119,20 @@ public void getManufacturerById() {
         assertEquals("Expected different manufacturer", Manufacturer.getName(), originalManufacturer.getName());
     }
 
+    @Test
+    public void getManufacturerByName() {
+        String name = "name";
+
+        Manufacturer originalManufacturer = new Manufacturer();
+        originalManufacturer.setName(name);
+        manufacturerDAO.save(originalManufacturer);
+
+        Manufacturer differentManufacturer = new Manufacturer();
+        differentManufacturer.setName("differentManufacturer");
+        manufacturerDAO.save(differentManufacturer);
+
+        Manufacturer Manufacturer = manufacturerDAO.findByName(name);
+        assertEquals("Expected different manufacturer", Manufacturer.getName(), originalManufacturer.getName());
```

Comment:
Nepouzivej nazvy promennych zacinajici velkym pismenem. Potom to vypada jako jako volani staticke metody.